non_iid_detection/requirements.txt

```
@@ -0,0 +1,7 @@
+# This notebook currently requires the latest developer version of cleanlab: pip install git+https://github.com/cleanlab/cleanlab.git 
+numpy==1.22.0
+pandas==1.4.1
+scipy==1.7.3
+scikit_learn==1.0.2
+torch==1.12.1
+torchvision==0.14.1
```

I get the following error on a fresh virtual environment:

```
ERROR: Cannot install -r non_iid_detection/requirements.txt (line 8) and torch==1.12.1 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested torch==1.12.1
    torchvision 0.14.1 depends on torch==1.13.1

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict
```

```suggestion
torch>=1.13.1
torchvision==0.14.1
```